tests/test_benchmark_spec.py

```diff
@@ -12,7 +14,13 @@ def _generate_test_path(p):
 
 
 def test_get_input_file_size():
-    assert _minimal_spec.get_input_file_size() == 161
+    size = _minimal_spec.get_input_file_size()
+    # Different operating systems use different file structures, which, as a result, leads to varying amounts of
+    # space being used for storing files.
+    if platform.system() == 'Windows':
+        assert size == 167
+    else:
+        assert size == 161
```

Comment:
Are we gaining anything by asserting the apparently platform-dependent size, as opposed to, e.g., just the name of the file?

Reply:
Discussed offline, comparing real size against expected size returned by the method `Path(...).stat().st_size()`